prediction_market_agent/agents/invalid_agent/deploy.py

```diff
@@ -0,0 +1,40 @@
+from prediction_market_agent_tooling.deploy.agent import DeployableTraderAgent
+from prediction_market_agent_tooling.deploy.betting_strategy import (
+    BettingStrategy,
+    KellyBettingStrategy,
+)
+from prediction_market_agent_tooling.gtypes import Probability
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+from prediction_market_agent_tooling.markets.data_models import ProbabilisticAnswer
+from prediction_market_agent_tooling.markets.markets import MarketType
+from prediction_market_agent_tooling.tools.is_invalid import is_invalid
+
+
+class InvalidAgent(DeployableTraderAgent):
+    bet_on_n_markets_per_run: int = 10
+
+    def verify_market(self, market_type: MarketType, market: AgentMarket) -> bool:
+        if market_type != MarketType.OMEN:
+            raise RuntimeError("This agent works only on Omen markets.")
```

Comment:
could you add a reason why in the error message 🥺

Reply:
Yes! Added